packages/vscode-extension/src/handlers.ts

```diff
@@ -681,8 +684,52 @@ export async function createNewEnvironment(args?: any[]): Promise<Result<Void, F
   return result;
 }
 
-export async function viewEnvironment(args?: any[]): Promise<Result<Void, FxError>> {
-  // todo add view logic
+export async function viewEnvironment(env: string): Promise<Result<Void, FxError>> {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
+    const envFilePath = path.join(
```

Comment:
Recommend to use `environmentManager.getEnvConfigPath` here.

Reply:
Fixed.